content/blog/2016/2016-08-30-ask-experts-demos.adoc

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: "Demos at Jenkins World 2016"
+tags:
+- jenkinsworld
+- event
+author: rtyler
+---
+
+image:/images/conferences/Jenkins-World_125x125.png[role=right]
+
+At this year's link:https://www.cloudbees.com/jenkinsworld/home[Jenkins World],
+our events officer link:https://github.com/alyssat[Alyssa] has been working to
+organize various activities in the "Open Source Hub" on the expo floor. One of
+the activities that I am personally most excited for are the Jenkins demos in
```

Comment:
This is tough, because the current wording suggests that it's one activity, but demos is plural. Argh my head! Maybe throwing away the explicit singular ("one of") would help.
"The activity that I am most excited about is the Jenkins demos in our 'open source theater.'" How about that?